front/src/views/Boxes/components/BoxesActionsAndTable.tsx

```diff
@@ -150,6 +162,12 @@ function BoxesActionsAndTable({
   } = useUnassignBoxesFromShipments();
 
   const onUnassignBoxesToShipment = useCallback(() => {
+    if (selectedBoxes.length === 0) {
+      createToast({
+        type: "warning",
+        message: `Please select a box to unassign from`,
+      });
+    }
```

Comment:
I couldn't test this, since the "Remove box from shipment" button only appears when at least one MarkedForShipment box is selected. We can still keep it in case the button appearance is changed.